src/test/java/com/epam/mjc/collections/list/ListSorterTest.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.mjc.collections.list;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class ListSorterTest {
+
+    @Test
+    public void sortEmptyList() {
+        ListSorter listSorter = new ListSorter();
+        ArrayList<String> sourceArrayList = new ArrayList<>();
+        ArrayList<String> expectedArrayList = new ArrayList<>();
+
+        listSorter.sort(sourceArrayList);
+        assertEquals(expectedArrayList, sourceArrayList);
+    }
+
+    @Test
+    public void sortNonEmptyList() {
+        ListSorter listSorter = new ListSorter();
+        ArrayList<String> sourceArrayList = new ArrayList<>();
```

Comment:
Why you use `ArrayList<>`? As I can see, method accepts `List<String>`. You can use List<> list = List.of(...)`